### DISCUSSION TITLE
第29期：征集周刊改版 idea💡 | 信息差——独立开发者出海周刊

### DISCUSSION THREAD

**Discussion by giscus** [weijunext/gapis.money#71] [General]
# 第29期：征集周刊改版 idea💡 | 信息差——独立开发者出海周刊

Knowledge is power, info-gap is money!「信息差——独立开发者出海周刊」是一个帮助独立开发者缩小信息差的技术周刊。

https://gapis.money/weekly/2024-12-09_029

**Comment by sir-bravo** [comment]
经常看阮一峰大大的周刊，从用户的角度谈谈感受。对比了一下。你们的内容感觉都差不多长，但你的翻一下就没了，我特意看了一下，他分享一个应用都截图了。另外这个界面功能比他的全，但是暗色调感觉有点压抑。目录一直在左边，显得内容板块不大气。

**Reply by weijunext** [reply]
建议很有用，感谢

**Comment by WymanY** [comment]
说实话阮大的周刊质量也越来越水了，有很大的推广的成分，我更希望找到一些高质量的东西，甚至特别好的教程。比如你推荐的这个： bknd，我就不确定质量咋样。主要这个周刊感觉还没形成权威性，只要看到你推荐的就是好的，就值得我深挖，目前还没有这种感受。 btw 这个 bknd 是推广吗？

**Reply by weijunext** [reply]
不是推广。这个网站只给中推圈里熟悉的朋友宣传过产品，比如youtube-dubbing、mkdirs。
